features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/webpack.config.js

```diff
@@ -118,5 +118,14 @@ module.exports = function (env) {
         const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
         config.plugins.push(new BundleAnalyzerPlugin());
     }
+    if (env && env.unused) {
+        const { UnusedFilesWebpackPlugin } = require('unused-files-webpack-plugin');
+
+        config.plugins.push(new UnusedFilesWebpackPlugin({
+            failOnUnused: process.env.NODE_ENV !== 'development',
+            patterns: ['source/src/**/*.jsx', 'source/src/**/*.js'],
+            ignore: ['babel.config.js', '**/*.txt', 'source/src/index.js'],
```

Comment:
i don't think u need to ignore `'**/*.txt'` since this will not match with any of the above patterns? @kavishkafernando  FYI 😃

Reply:
ack.